qa/restart-upgrade/src/test/resources/processor/IndexMappings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "settings": {
+    "index": {
+      "knn": true,
+      "knn.algo_param.ef_search": 100,
+      "refresh_interval": "30s",
+      "default_pipeline": "%s"
+    },
+    "number_of_shards": 6,
+    "number_of_replicas": 0
+  },
+  "mappings": {
+    "properties": {
+      "passage_embedding": {
+        "type": "knn_vector",
+        "dimension": 768,
+        "method": {
+          "name": "hnsw",
+          "space_type": "l2",
+          "engine": "lucene",
+          "parameters": {
```

Comment:
I think defaults are fine for engine params, for neural search it not that important